roles/openshift_health_checker/library/etcdstats.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/python
+'''
+Ansible module that determines if the size of paths in an etcd cluster exceed a given limit.
+'''
+
+from ansible.module_utils.basic import AnsibleModule
+
+IMPORT_EXCEPTION = None
+
+try:
+    import etcd
+# pylint: disable=invalid-name
+except ImportError as IMPORT_EXCEPTION:
+    # Allows for test mocking
```

Comment:
I thought it might be good to leave a reason here explaining why we catch an import exception, since it is something that is not really done in any other check